examples/discrete-id3.rb

```diff
@@ -18,17 +28,31 @@
 dec_tree = DecisionTree::ID3Tree.new(attributes, training, 1, :discrete)

```

Comment:
discrete-id3.rb:28:in `<main>': undefined local variable or method `attributes' for main:Object (NameError)